airbyte-integrations/connectors/source-google-sheets/google_sheets_source/models/definitions/spreadsheet.yaml

```diff
@@ -0,0 +1,65 @@
+---
+"$schema": http://json-schema.org/draft-07/schema#
```

Comment:
is there a reason it's particularly helpful to define these as json schema as opposed to using pojos (or whatever the python equivalent is)? i'm realizing i haven't been as thoughtful as i should have about this in the past, but it seems like if we are not planning to serialize these structs (which i don't think we are), adding the json schema layer is more trouble than its worth and might bring in some incidental complexity. if there's some other benefit you're getting then go for it.

Reply:
switched this to use only PyaDantic models. So basically it's like autogenerated code without the autogeneration. We'd need to do it manually. Couldn't find many great ways to do this otherwise.